pkg/provisioner/fake_nodeprovisioner.go

```diff
@@ -104,3 +105,7 @@ func (fake *FakeNodeProvisioner) SetMountCheckError(target string, err error) {
 
 	fakeMounter.SetMountCheckError(target, err)
 }
+
+func (fake *FakeNodeProvisioner) SetMounter(mounter *mount.SafeFormatAndMount) {
```

Comment:
Where is this used?

Reply:
`fake_azuredisk_v2.go` line: 165. Used to set the mounter through the fakeDriverV2.